### ISSUE TITLE
@Validate eclipse auto-import bug

### ISSUE THREAD

**Issue by nicholas22** [peichhorn/lombok-pg#44]
When using the @Validate annotation and some other one beneath it, the one beneath it is not recognised when fixing imports. Example:

```
  @Validate
  @SneakyThrows
  public void test(@NotNull final String name) {
      System.out.println("Hi "+name);
  }
```

Workaround:

```
  @SneakyThrows
  @Validate
  public void test(@NotNull final String name) {
      System.out.println("Hi "+name);
  }
```

Fixing import works, I guess due to @SneakyThrows is above and as such not within the @Validate annotation scope... This has echoes of the @Function auto-import bug https://github.com/peichhorn/lombok-pg/issues/21

**Comment by peichhorn** [comment]
fixed with the upgrade to lombok 0.10.3 including the usability fixes provided by Jappe van der Hel
 see issue #49

I will let you now when I upload a new edge release.

**Comment by nicholas22** [comment]
Tested and working on my end
